### PR TITLE
[tests] Update test-pwm.js for bug fix

### PR DESCRIPTION
**Pull request by cuiyanx**
Signed-off-by: Cui, Yan yanx.cui@intel.com

### CODE REVIEW

File: tests/test-pwm.js [hunk 1]
```diff
@@ -72,13 +71,12 @@ msTimer = setInterval(function () {
...
         msFalse = msFalse + 1;
     }
-
-    msCount = msCount + 1;
     // set 50 ms but 60 ms actually and time precision 20 ms for zephyr 1.6
 }, 50);
 
 setTimeout(function () {
-    assert(msTrue === 16 && msFalse === 34 && msCount === 50,
+    // 16 = 0.33 < 0.3333 < 0.34 = 17
```
Comment (brianjjones) [inline tests/test-pwm.js]: Is this intended as a comment, or is it old code?

Comment (cuiyanx) [inline tests/test-pwm.js]: This is a comment in the program, not old code.

**Comment by brianjjones** [comment]
LGTM +1

**Comment by grgustaf** [comment]
+1

**Pull request merged**